data-loader/cli/src/main/java/com/scalar/db/dataloader/cli/command/dataexport/ExportCommandOptions.java

```diff
@@ -144,4 +145,11 @@ public class ExportCommandOptions {
       description = "Size of the data chunk to process in a single task (default: 200)",
       defaultValue = "200")
   protected int dataChunkSize;
+
+  @CommandLine.Option(
+      names = {"--mode", "-sm"},
```

Comment:
Why `-sm`?

Reply:
@brfrn169 san,
`-m` is already  used by the `--include-matadata` input as this was already present
```
@CommandLine.Option(
      names = {"--include-metadata", "-m"},
      description = "Include transaction metadata in the exported data (default: false)",
      defaultValue = "false")
  protected boolean includeTransactionMetadata;
  ```
  I didn't want to change the existing value as it has been there from the always  and there may be users who use it.
  Then also the `--mode in import is `-m` (which also was added in latest change)
  Should I change the mode to `-m` and include metadata to something else like  `-im` ?

Reply:
@brfrn169 san,
After discussing with Yves san, I have set `-m` for mode and `-im` to inlcude-metadata to make it consistent with import.
Thank you.